src/grasshopper/lib/fixtures/__init__.py

```diff
@@ -531,3 +564,87 @@ def type_check_list_of_strs(list_of_strs):
             all_strs = all_strs and type(s) == str
         check_passed = all_strs
     return check_passed
+
+
+def _get_composite_weighted_user_classes(
+    full_scenarios_list, composite_scenario_contents
+):
+    """Generate a dictionary of journey classes with their associated weights."""
+    weighted_user_classes = {}
+    child_scenario_specs = _get_child_scenario_specs(
+        full_scenarios_list, composite_scenario_contents
+    )
+    for child_scenario_spec in child_scenario_specs:
+        base_journey_class = _import_class_with_journey(
+            absolute_file_path=f"{os.getcwd()}/"
```

Comment:
i think we either want to use os.path.join or the new file path object, which I'm blanking on the name of, here rather than calculating string directly.

Reply:
changed